tests/TestLaunchInventory.py

```diff
@@ -1,21 +1,24 @@
 import sys
+
 sys.path.append('.')
 from unittest import TestCase
 from inventory import parse_params
 import os
 import unittest
+import logging
+
+logger = logging.getLogger('test-logger')
 
 
 class TestLaunchExporter(TestCase):
```

Comment:
Shouldn't we add
`-v`
`-vv`
and check if loglevels are set as expected?

Reply:
Sure. I will find a way.